PurchasesCoreSwift/Purchasing/ProductsManager.swift

```diff
@@ -51,7 +51,7 @@ extension ProductsManager: SKProductsRequestDelegate {
 
     func productsRequest(_ request: SKProductsRequest, didReceive response: SKProductsResponse) {
         queue.async { [self] in
-            NSLog("products request received response")
+            Logger.info("products request received response")
```

Comment:
Should this be debug?

Reply:
I'm honestly not sure which is best here. I was kind of torn because of stuff like #250. I might improve the message a bit so that it's more useful.

Reply:
I should also make it consistent with line 70, where I'm using debug, so that either they're both debug or both info